vimrc

```diff
@@ -186,6 +186,10 @@ xnoremap <expr> p 'pgv"'.v:register.'y'
 
 imap <C-L> <SPACE>=><SPACE>
 
+" ========= Map jk to switch to command mode ========
+
+imap jk <Esc>
```

Comment:
+1.  More background on this: http://learnvimscriptthehardway.stevelosh.com/chapters/05.html